src/Uno.UITest.Helpers/AppInitializer.cs

```diff
@@ -268,7 +268,7 @@ private static IApp CreateiOSApp(bool alreadyRunningApp)
 
 			var app = alreadyRunningApp
 				? iOSConfig.ConnectToApp()
```

Comment:
We are passing in `mode` here which defaults to `Clear` but we were previously passing `DoNotClear` specifically to the `StartApp` call.